pkg/abap/aakaas/targetVector_test.go

```diff
@@ -171,7 +171,7 @@ func TestTargetVectorPublish(t *testing.T) {
 		mc := abapbuild.NewMockClient()
 		mc.AddData(AAKaaSHead)
 		mc.AddData(AAKaaSTVPublishTestPost)
-		errInitConn := conn.InitAAKaaS("", "dummyUser", "dummyPassword", &mc)
+		errInitConn := conn.InitAAKaaS("", "dummyUser", "dummyPassword", &mc, "")
```

Comment:
These are the only two tests? You don't want a test where this hash is actually set and you check the value?

Reply:
now added additional tests for testing hash to be added to http headers (as well as all headers in general)